gui/default/syncthing/core/syncthingController.js

```diff
@@ -554,9 +572,19 @@ angular.module('syncthing.core')
 
                 data = data.connections;
                 for (id in data) {
+                    $scope.idToWebAddress[id] = "";
                     if (!data.hasOwnProperty(id)) {
                         continue;
                     }
+                    var port = $scope.findDevice(id).remoteGUIPort;
+                    var isNotRelayConnection = !data[id].type.includes("relay");
+                    if ($scope.showRemoteGUI && data[id].address !== "" && isNotRelayConnection) {
+                        var newAddress = `http://${replaceAddressPort(data[id].address, port)}`
```

Comment:
I don't think this fancy string interpolation is very compatible across many browsers.